src/components/ebay-calendar/examples/default.marko

```diff
@@ -0,0 +1 @@
+<ebay-calendar ...input/>
```

Comment:
All events should be emitted up one level from here so we can show them on storybook.